index.md

```diff
@@ -11,16 +11,16 @@ Before you get started
 
 ###  What is Rails?
 
-Rails is a framework for building web applications and it uses the Ruby programming language. The important thing that differentiates a web application from a web-site is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
+Rails is a framework for building web applications, and it uses the Ruby programming language. The important thing that differentiates a web application from a website is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
 
-Rails is also open source means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensure that it continues to grow and develop with technology, anyone can contribute to it's development. This also means that Rails will always be free to use.
+Rails is also open source, which means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensures that it continues to grow and develop with technology, anyone can contribute to its development. This also means that Rails will always be free to use.
 
-As you work through the tutorial, don't stress if you don't understand every detail perfectly. This technology is complex and it takes a lot of practice to start feeling like you truely understand how it all works. 
+As you work through the tutorial, don't stress if you don't understand every detail perfectly. This technology is complex, and it takes a lot of practice to start feeling like you truly understand how it all works.
 
 ### Prerequisites
 
-1. Before building anything with Rails you'll need to get your computer set up. Check out our [install instructions]({{ site.baseurl }}/installation.html) to be sure you have everything set up and ready to go.
+1. Before building anything with Rails, you'll need to set up your computer. Check out our [install instructions]({{ site.baseurl }}/installation.html) to be sure you have everything set up and ready to go.
```

Comment:
The [Chicago Manual of Style](http://www.chicagomanualofstyle.org/qanda/data/faq/topics/Commas/faq0030.html) is less clear.
![img_20170303_002111](https://cloud.githubusercontent.com/assets/2002467/23539192/6c652f06-ffa7-11e6-9b40-70e6ac0f2c7e.jpg)

(The citation numbers are different because they are citing 16th edition, and I have the 15th edition.)

The Purdue Online Writing Lab suggests that you need to use a comma if the introductory phrase is longer than 4 words. The [Microsoft Manual of Style](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=1&ved=0ahUKEwi2t9aNzLnSAhUT24MKHWbjBnQQFggaMAA&url=https%3A%2F%2Feucalyptus.atlassian.net%2Fwiki%2Fdownload%2Fattachments%2F76611622%2Fmicrosoft_manual_of_style_fourth_edition.pdf%3Fversion%3D2%26modificationDate%3D1424379604164%26api%3Dv2&usg=AFQjCNH4LGu0suDPoWJLSCyhI_3JAdT8Ww&sig2=Ulg9MLpNE65mdnwzPR4FLA) says that you should always use the commas for clarity. (I suspect that is because technical manuals are notoriously awful to read, and ESL readers benefit from the additional consistency and clarity.)